modules/oxide/src/less/skins/content/dark/content.less

```diff
@@ -85,3 +90,62 @@ code {
     padding-right: 1rem;
   }
 }
+//
+// Comments
+//
+
+@document-comment-background-color: contrast(@content-text-color, lighten(@color-active, 20%), mix(@color-active, @color-black, 20%)); 
+@document-comment-active-background-color: @document-comment-background-color;
+@document-comment-outline-color: @color-active;
+@document-comment-active-outline-color: @color-active;
+
+@document-comment-outline-width: 3px;
+
+.tox-comments-visible {
+  --text-color: @content-text-color;
+  --color-active: @color-active;
+  --comment-color: contrast(@content-text-color, lighten(@color-active, 20%), mix(@color-active, @color-black, 20%));
+  --content-background-color: @content-background-color;
+
+  .tox-comment[contenteditable="false"],
+  span.tox-comment img,
+  span.tox-comment > audio,
+  span.tox-comment > video,
+  span.tox-comment span.mce-preview-object {
+    &:not([data-mce-selected]) {
+      outline: @document-comment-outline-width solid @document-comment-outline-color;
+    }
+  }
+
+  .tox-comment[contenteditable="false"][data-mce-annotation-active="true"] {
+    &:not([data-mce-selected]) {
+      outline: @document-comment-outline-width solid @document-comment-active-outline-color;
+    }
+  }
+
+  span.tox-comment[data-mce-annotation-active="true"] {
+    & img,
+    & > audio,
+    & > video,
+    & span.mce-preview-object {
+      &:not([data-mce-selected]) {
+        outline: @document-comment-outline-width solid @document-comment-active-outline-color;
+      }
+    }
+  }
+
+  span.tox-comment:not([data-mce-selected]) {
+    background-color: @document-comment-background-color;
+    outline: none;
```

Comment:
I would expect only color overrides in this skin file. Is this needed to be set here isn't the base skin setting this already?